xbmc/cores/VideoPlayer/VideoRenderers/DebugRenderer.h

```diff
@@ -38,6 +38,11 @@ class CDebugRenderer
     void CreateSubtitlesStyle();
 
   private:
+    // Implementation of Observer
```

Comment:
We use the notify/observer to inform the overlayrenderer (since it has to react to any of the settings being monitored) and the settings callback to perform individual actions on individual settings change in this component right?